site/source/docs/porting/connecting_cpp_and_javascript/embind.rst

```diff
@@ -248,6 +240,19 @@ a C++ object is no longer needed and can be deleted:
         }
     }
 
+Automatic management
```

Comment:
Does this need a noun maybe?  Automatic lifetime management?  Automatic memory management?